<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>304</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>304</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QLineEdit" name="importPathBox"/>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="importButton">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSpinBox" name="sizeBox">
     <property name="prefix">
      <string>Matrix Size: </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="selectionBox1">
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="selectionBox2"/>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="mathBox">
     <item>
      <property name="text">
       <string>Addition (+)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Subtraction (-)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multiplication (*)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inversion</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Transposition</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="calculateButton">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="exportButton">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTableWidget" name="matrixTable1">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QTableWidget" name="matrixTable2">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QTableWidget" name="matrixTable3">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
